.github/workflows/release.yml

```diff
@@ -1,31 +0,0 @@
-name: Release CI
```

Comment:
i would suggest rather than deleting this workflow, we should disable it from [Github](https://github.com/edx/frontend-component-authn-edx/actions/workflows/release.yml)
 
<img width="1664" alt="image" src="https://github.com/edx/frontend-component-authn-edx/assets/42172960/b480d0d8-a7df-4bd7-942d-19298a01a0cf">

Reply:
I disabled it. Thanks

https://github.com/edx/frontend-component-authn-edx/actions/workflows/release.yml